src/dotnet/Plugin.props

```diff
@@ -1,7 +1,7 @@
 ﻿<Project>
 
   <PropertyGroup>
-    <SdkVersion>2021.3.1</SdkVersion>
+    <SdkVersion>2021.4.0</SdkVersion>
```

Comment:
What SDK version is this supposed to be? Currently `2021.3.2` is highest.

Reply:
oops. i thought that was the version number